### DISCUSSION TITLE
When using the file clipboard, problems occur with the zip file.

### DISCUSSION THREAD

**Discussion by kimcoldwater** [FreeRDP/FreeRDP#8904] [Q&A]
Hi, I created a client and implemented a file clipboard.

Problems occur when the zip file is moved to remote session -> local (mac).

Attempting to decompress will result in a warning statement  Improper File Type and Format and will not be able to decompress.

Local (mac) -> There is no problem with zip movement to remote session, but it only occurs when moving from remote session to local (mac).

The 'tar.gz' compression file and other files are moved normally.

Is there a possible cause of this problem only for zip files?

**Comment by akallabeth** [comment]
there is no notion of `file type` in the file clipboard.
you simply have an error while copying, check with e.g. `sha256sum` on server and client, the hashes will not match.
